client/src/components/PortfolioFilters.tsx

```diff
@@ -101,92 +104,111 @@ export const PortfolioFilters = React.memo(
+        >
+          {" "}
+          <button
+            aria-pressed={rsunitslatestActive}
+            onClick={updateRsunitslatest}
+            className="filter filter-toggle"
+          >
+            <div className="checkbox">{rsunitslatestActive && <CheckIcon />}</div>
+            <span>
               <Trans>Rent Stabilized Units</Trans>
+            </span>
+          </button>
+          <FilterAccordion
+            title={i18n._(t`Landlord`)}
+            subtitle={i18n._(t`Officer/Owner`)}
+            infoOnClick={() => setShowOwnerModal(true)}
+            isMobile={isMobile}
+            isActive={ownernamesActive}
+            isOpen={ownernamesIsOpen}
+            setIsOpen={setOwnernamesIsOpen}
+            selectionsCount={filterContext.filterSelections.ownernames.length}
+            className="ownernames-accordion"
+          >
+            <Multiselect
+              options={ownernamesOptions.map((value: any) => ({ name: value, id: value }))}
+              selectedValues={ownernamesSelections}
+              displayValue="name"
+              placeholder={i18n._(t`Search`) + `... (${ownernamesOptions.length})`}
+              onApply={onOwnernamesApply}
+              onFocusInput={() => helpers.scrollToBottom(".mobile-wrapper-dropdown")}
+              infoAlert={OwnerInfoAlert}
+              avoidHighlightFirstOption={true}
+            />
+          </FilterAccordion>
+          <FilterAccordion
+            title={i18n._(t`Building Size`)}
+            subtitle={i18n._(t`Number of Units`)}
+            isMobile={isMobile}
+            isActive={unitsresActive}
+            isOpen={unitsresIsOpen}
+            setIsOpen={setUnitsresIsOpen}
+            className="unitsres-accordion"
+          >
+            <MinMaxSelect
+              options={filterContext.filterOptions.unitsres}
+              onApply={onUnitsresApply}
+              onFocusInput={() => helpers.scrollToBottom(".mobile-wrapper-dropdown")}
+            />
+          </FilterAccordion>
+          <FilterAccordion
+            title={i18n._(t`Zipcode`)}
+            isMobile={isMobile}
+            isActive={zipActive}
+            isOpen={zipIsOpen}
+            setIsOpen={setZipIsOpen}
+            selectionsCount={filterContext.filterSelections.zip.length}
+            className="zip-accordion"
+          >
+            <Multiselect
+              options={zipOptions.map((value: any) => ({ name: value, id: value }))}
+              selectedValues={zipSelections}
+              displayValue="name"
+              placeholder={i18n._(t`Search`) + `... (${zipOptions.length})`}
+              onApply={onZipApply}
+              onFocusInput={() => helpers.scrollToBottom(".mobile-wrapper-dropdown")}
+              avoidHighlightFirstOption={true}
+            />
+          </FilterAccordion>
+        </FiltersWrapper>
+        {[rsunitslatestActive, ownernamesActive, unitsresActive, zipActive].includes(true) && (
```

Comment:
could this just be a big "or" statement? `(rsunitslatesActive || ownernamesActive || ...)` probably not a huge diff but might be marginally more performant than the includes

Reply:
ah good to know, yeah that is just simplier all around not sure what I was doing here.